jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -1488,50 +1515,51 @@ YUI.add('juju-env-go', function(Y) {
       @return {undefined} Sends a message to the server only.
     */
     _add_unit: function(service, numUnits, toMachine, callback) {
```

Comment:
My understanding is that passing 42 or 2/lxc/1 should just work: the parsed placement is like {Scope: '#', Directive: '42'} and that's it. I QAed with top level machines, we should try with containers during QA.